src/server/services/wms/qgswmsrenderer.cpp

```diff
@@ -3452,7 +3452,7 @@ namespace QgsWms
     mTemporaryLayers.clear();
   }
 
-  QPainter *QgsRenderer::layersRendering( const QgsMapSettings &mapSettings, QImage &image ) const
+  QPainter *QgsRenderer::layersRendering( const QgsMapSettings &mapSettings, QImage *image ) const
```

Comment:
Can we not keep the reference instead, and just dereference the unique_ptr when calling the method. It's more explicit about who has ownership of the image object

Reply:
@troopa81 I have a change, what do you think?